change/@fluentui-react-conformance-35e35a06-cb99-40ac-9617-72e443ec3109.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

```suggestion
  "type": "major",
```

shouldn't this be a major?

---

Guessing since `react-conformance` is still in pre-release version 0.x.y, minor bumps act as major?

---

yeah, major is not allowed ( its not the best DX from the tool we use 💁 )